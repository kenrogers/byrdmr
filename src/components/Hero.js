import React, { Component } from "react";
import styled from "styled-components";

import drone from "../img/Drone.png";

class Hero extends Component {
  render() {
    const Hero = styled.div`
      min-height: 100vh;
      background: #f5f0ec;
      width: 100%;
      display: flex;
      flex-direction: column;
      justify-content: space-between;
      align-items: center;
      padding-top: 120px;
      img {
        max-width: 429px;
        margin: auto;
        display: block;
      }
      h1 {
        color: #fff;
        font-size: 235px;
        text-align: center;
        margin: 0;
      }
      .quote {
        margin-bottom: 3rem;
      }
      p {
        text-align: center;
        font-size: 18px;
        opacity: 0.2262;
        text-transform: uppercase;
        letter-spacing: 1.29px;
        margin: 0;
        &.credit {
          text-transform: none;
          letter-spacing: 1px;
          font-size: 14px;
        }
      }
    `;

    return (
      <Hero>
        <img src={drone} alt="ByrdMR Drone" />
        <h1>F-210</h1>
        <div className="quote">
          <p>"The fastest drone on the planet"</p>
          <p className="credit">&ndash; The Wirecutter</p>
        </div>
      </Hero>
    );
  }
}

export default Hero;
