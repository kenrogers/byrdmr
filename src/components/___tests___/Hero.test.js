import React from 'react'
import Hero from '../Hero'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

describe('<Hero/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Hero />)
    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
