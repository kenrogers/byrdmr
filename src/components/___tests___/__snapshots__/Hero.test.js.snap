// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero/> renders and matches the snapshot 1`] = `
<styled.div>
  <img
    alt="ByrdMR Drone"
    src="Drone.png"
  />
  <h1>
    F-210
  </h1>
  <div
    className="quote"
  >
    <p>
      "The fastest drone on the planet"
    </p>
    <p
      className="credit"
    >
      â€“ The Wirecutter
    </p>
  </div>
</styled.div>
`;
